settings.ui

```diff
@@ -10,7 +10,7 @@
     <x>0</x>
     <y>0</y>
     <width>660</width>
-    <height>352</height>
+    <height>384</height>
```

Comment:
Was this change intentional? If so, why was it needed? Is the window size too small for the sample path?

Reply:
I'm not able to make it smaller, seems like it auto adjusted to make room for the EEPROM settings, I don't know enough about QT Designer currently to confirm that though.

Reply:
Try it outside the designer and see if it causes any problems (edit the XML directly). I'm not against changing the window's default size, but should be handled outside this PR.

Reply:
I think this is fine. You'd expect the size (height) to grow when adding new widgets.